components/build-templates/production/e2e-registryredhatio-pull-secret.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: registryredhatio-pull-secret
```

Comment:
will it be good to name it in a more user friendly way like `registry-redhat-io-pull-secret`

Reply:
updated